### PR TITLE
Doc: update create_guest_image and launch_td_guest

### PR DESCRIPTION
**Pull request by ruomengh**
1. Modify create_guest_image to use rhel-8 as default for external users
2. Revise invalid URL

Signed-off-by: Hao, Ruomeng <ruomeng.hao@intel.com>

### CODE REVIEW

File: doc/create_guest_image.md [hunk 1]
```diff
@@ -11,9 +11,8 @@ For direct boot, any type of cloud image
 | MBR based Legacy | [CentOS Cloud Image](https://cloud.centos.org/centos/8-stream/) | Support direct boot for TDX guest |
 | GPT based EFI | [Ubuntu Cloud Image](https://cloud-images.ubuntu.com/) | Support both direct and EFI grub boot for TDX guest |
 
-This chapter use `CentOS Stream` as example base distro. The default cloud image
-for CentOS Stream does not support EFI schema, so need create customized EFI
-based CentOS image via [TDX Guest Image Tool](https://github.com/intel/tdx-tools/tree/main/build/centos-stream-8/guest-image).
+The default cloud image for RHEL and CentOS Stream does not support EFI schema, so it needs
```
Comment (jialeif) [inline doc/create_guest_image.md:14]: default rhel cloud image supports EFI schema.

**Comment by jialeif** [comment]
It's better to keep original c8s example.
today one customer requires rhel example, in future another customer requires c8s example.

File: doc/create_guest_image.md (hunk 1 above)
Comment (ruomengh) [reply doc/create_guest_image.md]: @jialeif Thanks for the comments. Please review again.

**Comment by ruomengh** [comment]
> It's better to keep original c8s example. today one customer requires rhel example, in future another customer requires c8s example.

Both c8s and rhel are added to the doc. User can find both rhel and c8s according to the doc.

**Review by jialeif — APPROVED** [review]
LGTM

**Pull request merged**